app/domain/authentication/authn_jwt/authenticator.rb

```diff
@@ -56,5 +56,25 @@ def new_token
         )
       end
     end
+
+    class Authenticator
+      # This delegates to all the work to the call method created automatically
+      # by CommandClass
+      #
+      # This is needed because we need `valid?` to exist on the Authenticator
+      # class, but that class contains only a metaprogramming generated
+      # `call(authenticator_input:)` method.  The methods we define in the
+      # block passed to `CommandClass` exist only on the private internal
+      # `Call` objects created each time `call` is run.
+      def valid?(input)
+        call(jwt_configuration:input[jwt_configuration] ,authenticator_input: input[autheticator_input])
+      end
+
+      def status(authenticator_status_input:)
+        Authentication::AuthnJwt::ValidateStatus.new.(
```

Comment:
Use .call(

I dont like .new.

Reply:
we still need the `.new` as we create a class. it should be `.new.call(` though

Reply:
Yes that what i meant

Reply:
I opened a new pr (https://github.com/cyberark/conjur/pull/2195) that adds a new API endpoint for jwt authenticator, please review it and add your comment if its still relevant